src/main.zig

```diff
@@ -334,3 +347,18 @@ pub fn main() !void {
         },
     }
 }
+
+var verbose: bool = builtin.mode == .Debug; // In debug mode, force verbose.
```

Comment:
Not a big fan of this global variable but I guess it's not possible to do without, at least if we want to keep using the std lib log function.

Reply:
Well, I think have a global var in the `main.zig` is not perfect, but ok.